test/test-ghe-backup.sh

```diff
@@ -463,9 +463,12 @@ begin_test "ghe-backup stores version when not run from a clone"
   # Make sure this doesn't exist
   rm -f "$GHE_REMOTE_DATA_USER_DIR/common/backup-utils-version"
 
-  mv "$ROOTDIR/.git" "$ROOTDIR/.gittmp"
-  ghe-backup
-  mv "$ROOTDIR/.gittmp" "$ROOTDIR/.git"
+  tmpdir=$TRASHDIR/ghe-backup-stores-version-$RANDOM
```

Comment:
Ended up using @sbryant's `mktemp -d` suggestion, that works with BSD and Linux `mktemp`.